### PR TITLE
Add temporary "freetype-config" workaround for PHP 7.3, 7.2, and 7.1 on buster

### LINKED ISSUE #865: Installing gd on buster results in freetype-config not found
Not using cached versions. 
I've tested the install on another device, same output.

Dockerfile:
```
FROM php:7.3.7-fpm
RUN apt-get update
RUN apt-get install -y --no-install-recommends \
        libfreetype6-dev \
        libjpeg62-turbo-dev \
        libpng-dev \
    && docker-php-ext-install -j$(nproc) iconv \
    && docker-php-ext-configure gd --with-freetype-dir=/usr/include/freetype --with-jpeg-dir=/usr/include/ \
    && docker-php-ext-install -j$(nproc) gd \
    && rm -r /var/lib/apt/lists/*

```
```
configure: error: freetype-config not found.
ERROR: Service 'php' failed to build: The command '/bin/sh -c apt update && apt install -y --no-install-recommends         libfreetype6-dev         libjpeg62-turbo-dev         libpng-dev     && docker-php-ext-install -j$(nproc) iconv     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/     && docker-php-ext-install -j$(nproc) gd     && rm -r /var/lib/apt/lists/*' returned a non-zero code: 1
```

The issue is present when using the latest version 7.3.7-fpm (buster)
The build succeeds when using 7.3.6-fpm (stretch)

TLDR solution for now
Use a tag with stretch instead of buster.

### PR DESCRIPTION
**Pull request by tianon**
See:

- https://github.com/docker-library/php/issues/865
- https://bugs.php.net/bug.php?id=76324
- https://github.com/php/php-src/pull/3632
- https://github.com/php/php-src/commit/2d03197749696ac3f8effba6b7977b0d8729fef3

This is not necessary for Debian Stretch (where `freetype-config` still exists) nor PHP 7.4+ (where `pkg-config` is used instead/in addition).

Closes #865

### CODE REVIEW

**Comment by tianon** [comment]
I verified this hack works as intended via the following patch to `drupal`:

```diff
diff --git a/8.7/apache/Dockerfile b/8.7/apache/Dockerfile
index 5c0d588..24654f5 100644
--- a/8.7/apache/Dockerfile
+++ b/8.7/apache/Dockerfile
@@ -1,7 +1,9 @@
 # from https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements
-FROM php:7.3-apache-stretch
+FROM php:7.3-apache-buster
 # TODO switch to buster once https://github.com/docker-library/php/issues/865 is resolved in a clean way (either in the PHP image or in PHP itself)
 
+RUN { echo '#!/bin/sh'; echo 'exec pkg-config "$@" freetype2'; } > /usr/local/bin/freetype-config && chmod +x /usr/local/bin/freetype-config
+
 # install the PHP extensions we need
 RUN set -eux; \
        \
```

(Which would otherwise fail without applying this workaround.)
